### PR TITLE
[13.0][MIG] project_milestone

### PR DESCRIPTION
**Pull request by davejames**

### CODE REVIEW

**Comment by davejames** [comment]
@patrickrwilson would you be able to review this please?

File: project_milestone/README.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+.. |maintainer-patrickrwilson| image:: https://github.com/patrickrwilson.png?size=40px
+    :target: https://github.com/patrickrwilson
+    :alt: patrickrwilson
+
+Current `maintainer <https://odoo-community.org/page/maintainer-role>`__:
+
+|maintainer-patrickrwilson| 
+
+This module is part of the `OCA/project <https://github.com/OCA/project/tree/12.0/project_milestone>`_ project on GitHub.
```
Comment (patrickrwilson) [inline project_milestone/README.rst:90]: I assume that all the references to 12.0 will be updated when the readme gets processed?

File: project_milestone/i18n/de.po [hunk 2]
```diff
@@ -0,0 +1,165 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* project_milestone
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
```
Comment (patrickrwilson) [inline project_milestone/i18n/de.po:7]: Same here, I think these references to 12.0 will be auto updated?

**Review by patrickrwilson — APPROVED** [review]
LGTM assuming that my comments where items are referencing 12.0 get automatically updated.

**Review by dreispt — APPROVED** [review]

**Pull request merged**